specification/maps/data-plane/Creator/preview/2022-09-01-preview/routeset.json

```diff
@@ -0,0 +1,360 @@
+        "operationId": "routesetDelete",
+        "x-ms-examples": {
+          "Delete an existing routeset": {
+            "$ref": "./examples/wayfind/Routeset_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeleteRoutesetId"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Routeset delete request completed successfully."
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/routesets/operations/{operationId}": {
+      "get": {
+        "description": "This API allows the caller to view the current progress of a routeset operation. The path is obtained from a call to the Create Routeset API.\n\n### Submit Operations Request\n\nTo view the current progress of a routeset operation, you will use a `GET` request with the `operationId` of the desired operation.\n\n### Operation Response\n\nWhile in progress, a `200-OK` http status code will be returned with no extra headers. If the operation succeeds, a `200-OK` http status code with Resource-Location header will be returned.",
+        "operationId": "routesetGetOperation",
+        "x-ms-examples": {
+          "Get the status of an operation which is still running": {
+            "$ref": "./examples/wayfind/Routeset_GetOperationStillRunning.json"
+          },
+          "Get the status of an operation which has finished successfully": {
+            "$ref": "./examples/wayfind/Routeset_GetOperation.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/OperationId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation is running or complete. If the operation was successful, use the Resource-Location header to obtain the path to the result.",
+            "schema": {
+              "$ref": "./tileset.json#/definitions/LongRunningOperationResult"
+            },
+            "headers": {
+              "Resource-Location": {
+                "type": "string",
+                "description": "If successful, a URI where details on the newly created resource can be found."
+              }
```

Comment:
This might not be a very standard LRO process. But OK if all map uses this pattern and your team write your own SDK.

Reply:
this is how the other creator services API's do it. It was covered with the review board.